src/components/StakingHierarchy.js

```diff
@@ -0,0 +1,465 @@
+            altogether, if that's not your thing.
+          </p>
+          <p>
+            Check out some solo staking tools, then get started staking on the
+            Prater Testnet over on the{" "}
+            <Link to="https://prater.launchpad.ethereum.org">
+              Staking Launchpad
+            </Link>
+            .
+          </p>
+          <ButtonLink
+            to="/staking/solo/"
+            onClick={() => {
+              trackCustomEvent({
+                eventCategory: `StakingHierarchy`,
+                eventAction: `Clicked`,
+                eventName: "clicked solo staking",
+              })
+            }}
+          >
+            More on solo staking
+          </ButtonLink>
+        </Content>
+      </Section>
+      <Section number="2">
+        <Ether>
+          <StyledEtherSvg size="90%" />
+        </Ether>
+        <Line />
+        <Header>
+          <h2>Staking as a service</h2>
+          <Pills className="subtext">
+            <p>Your 32 ETH</p>
+            <p>Your validator keys</p>
+            <p>Entrusted node operation</p>
+          </Pills>
+        </Header>
+        <Glyph>
+          <SaasGlyph />
+        </Glyph>
+        <Content>
+          <p>
+            Running your own hardware is always recommended if possible for the
+            strength and health of the network, but we recognize this isn't
+            always reasonable for all users.
+          </p>
+          <p>
+            If you don't want or don't feel comfortable dealing with hardware,
+            but still want to stake your 32 ETH, staking-as-a-service options
+            allow you to delegate the hard part, while you earn native block
+            rewards.
+          </p>
+          <p>
+            These options usually walk you through creating a set of validator
+            credentials, uploading your signing keys to them, and depositing
+            your 32 ETH. This allows the service to validate on your behalf.
+          </p>
+          <p>
+            This method of staking requires a certain level of trust in the
+            provider. To limit counter-party risk, the keys to withdrawal your
+            ETH are usually kept in your possession.
+          </p>
+          <ButtonLink
+            onClick={() => {
+              trackCustomEvent({
+                eventCategory: `StakingHierarchy`,
+                eventAction: `Clicked`,
+                eventName: "clicked staking as a service",
+              })
+            }}
+            to="/staking/saas/"
+          >
+            More on staking as a service
+          </ButtonLink>
+        </Content>
+      </Section>
+      <Section number="3">
+        <Ether>
+          <StyledEtherSvg size="80%" />
+        </Ether>
+        <Line />
+        <Header>
+          <h2>Pooled staking</h2>
+          <Pills className="subtext">
+            <p>Stake what you can</p>
```

Comment:
"Stake what you can" seems like a strange value prop. Perhaps we rephrase this to "Low barrier to entry"? Or "Low ETH requirements"?

Reply:
How about "Stake any amount?"... this isn't strictly true for every product in this category, but is true for the aggregate as there are multiple options to stake any amount of ETH.